includes/abstracts/abstract-wc-log-handler.php

```diff
@@ -0,0 +1,141 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+if ( ! class_exists( 'WC_Logger' ) ) {
+	include_once( dirname( dirname( __FILE__ ) ) . '/class-wc-logger.php' );
+}
+
+/**
+ * Abstract WC Log Handler Class
+ *
+ * @version        1.0.0
+ * @package        WooCommerce/Abstracts
+ * @category       Abstract Class
+ * @author         WooThemes
+ */
+abstract class WC_Log_Handler {
+
+	/**
+	 * Minimum log level this handler will process.
+	 *
+	 * @var int 0-8 minimum level severity for handling log entry.
+	 * @access private
+	 */
+	protected $threshold;
+
+	/**
+	 * Log levels by severity.
+	 *
+	 * @var array
+	 * @access private
+	 */
+	protected static $log_levels = array(
+		WC_Logger::DEBUG     => 0,
```

Comment:
Why is it best for the level → number mapping to live in the abstract class, while the string constants live in `WC_Logger`?